src/app/components/AllowanceMenu/index.tsx

```diff
@@ -120,11 +123,11 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
             <div className="w-60">
               <DualCurrencyField
                 id="budget"
-                label="New budget"
+                label={t("new_budget_label")}
```

Comment:
Could this be `new_budget.label` and below `new_budget.hint`?

Reply:
I was actually about to use them but then thought all other components don't extend further as in nested objects in translations so I thought of keeping it simple without nesting...

Reply:
:D Let's nest it